packages/core/src/handlers/http.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application} from '../application';
+import {ServerRequest, ServerResponse} from 'http';
+import {Handler} from '../internal-types';
+
+export function httpHandler(app: Application): Handler {
+  return async (req: ServerRequest, res: ServerResponse): Promise<void> => {
+    await app.handleHttp(req, res);
+  };
```

Comment:
However, based on the example usage in [sandbox](https://github.com/strongloop/loopback-next-sandbox/blob/f2b7777d2338b9aadebd5d887bb1d24330f4201a/ts/src/index.ts#L9):

```ts
http.createServer(handleHttp(app, {httpsRedirect: true})).listen(80);
https.createServer(handleHttps(app)).listen(443);
```

I think we need to modify `httpHandler` to take care of unhandled promise rejections inside application's `handleHttp` function. At minimum, the response should be ended with 500 Internal Error. See how `Server.ts` is implements error handler [here](https://github.com/strongloop/loopback-next/blob/041d1fdc6cb3a139435b40147ed7e9da4cbb16c7/packages/core/src/server.ts#L35-L52).

Alternatively, we can change `app.handleHttp` to take care of unhandled promises, in which case the signature of `Handler` function should be changed to specify an empty (void) return type.

```ts
export type Handler = (request: ServerRequest, response: ServerResponse) => void;
```